spa/tsconfig.json

```diff
@@ -2,10 +2,9 @@
   "compilerOptions": {
     "target": "ES2020",
     "useDefineForClassFields": true,
-    "lib": ["ES2020", "DOM", "DOM.Iterable"],
```

Comment:
I think adding a ES2022.Error is pretty safe. 
https://caniuse.com/mdn-javascript_builtins_error_cause

Reply:
Have to add this to support the `{ cause }` in Error constructor.